index.js

```diff
@@ -37,11 +37,27 @@ function handleError(error) {
   process.exitCode = 1;
 }
 
+// The following code block overrides commander's outputHelp()
+delete program.outputHelp;
+program.outputHelp = function (cb) {
+  printHeader();
+  if (!cb) {
+    // eslint-disable-next-line no-param-reassign
+    cb = function (passing) {
+      return passing;
+    };
+  }
+  process.stdout.write(cb(this.helpInformation()));
+  this.emit('--help');
+};
+module.exports = program;
```

Comment:
Is this line needed? I don't think anybody outside `index.js` uses the `program` variable inside here.